docs/Index.tsx

```diff
@@ -220,7 +220,9 @@ class App extends React.Component<any, any> {
                         {component: BannerExamples, componentName: 'Banner'},
                         {component: SlideYExamples, componentName: 'SlideY'},
                         {component: StatusCardExamples, componentName: 'StatusCard'},
-                    ].map((component) => <ExampleWrapper key={component.componentName} componentName={component.componentName} component={component.component} />)}
+                    ]
+                        .sort((componentA, componentB) => componentA.componentName.toLowerCase().localeCompare(componentB.componentName.toLowerCase()))

```

Comment:
Wrapping this into a `const caseInsensitiveCompare = (a, b) => a.toLowerCase().localeCompare(b.toLowerCase())` would help a little bit on the readability:

`.sort((componentA, componentB) => caseInsensitiveCompare(componentA.componentName, componentB.componentName))`